aospy/calc.py

```diff
@@ -322,6 +321,53 @@ def _to_desired_dates(self, arr):
                           self.end_date_xray, self.months, indices=False)
         return arr.sel(time=times)
 
+    def _add_grid_attributes(self, ds, n=0):
+        """Add model grid attributes to a dataset"""
+
+        grid_attrs = {
+            'lat':         ('lat', 'latitude', 'LATITUDE', 'y', 'yto'),
+            'lat_bounds':  ('latb', 'lat_bnds', 'lat_bounds'),
+            'lon':         ('lon', 'longitude', 'LONGITUDE', 'x', 'xto'),
+            'lon_bounds':  ('lonb', 'lon_bnds', 'lon_bounds'),
+            'level':       ('level', 'lev', 'plev'),
+            'sfc_area':    ('area', 'sfc_area'),
+            'zsurf':       ('zsurf',),
+            'land_mask':   ('land_mask',),
+            'pk':          ('pk',),
+            'bk':          ('bk',),
+            'phalf':       ('phalf',),
+            'pfull':       ('pfull',),
+            }
+
+        for name_int, names_ext in grid_attrs.items():
+            ds_coord_name = set(names_ext).intersection(set(ds.coords))
+            if ds_coord_name:
+                # Check if coord is in dataset already.
+                # If it is, then rename it so that it has 
+                # the correct internal name.
+                ds = ds.rename({list(ds_coord_name)[0] : name_int})
+                ds = ds.set_coords(name_int)
+                if not self._strip_undefined_coords(ds[name_int]).equals(getattr(self.model[n], name_int)):
+                    print("Warning: Model coordinates for '{}' do not match those in Run".format(name_int))
```

Comment:
Let's use the builtin `warnings` module (which you introduced me to!) here rather than just printing.  (At some point, I'd like to start using the logging module too to more methodically and systematically log runtime status, metadata of calculations, etc.)